packages/api/__tests__/index.js

```diff
@@ -290,11 +291,13 @@ describe('/api/profile/createUser', () => {
         json: true,
         resolveWithFullResponse: true,
       }).then((res) => {
-        const cooky = res.headers['set-cookie'][0].substr(6, 200);
+        const rawcookies = res.headers['set-cookie'];
+        const cookieWithHeaders = new Cookie(rawcookies[0]);
```

Comment:
This is progress, but what if backend changes to set multiple cookies, and the token is no longer the first cookie in rawcookies? Better to not rely on non-explicit things like position.

Reply:
ok. but will there be any other cookie apart from the login Token while logging in the user?

Reply:
Simple solution:
`rawcookies.find(() => { // Your code })`